src/lib.rs

```diff
@@ -76,6 +78,57 @@ struct AddressLineInfo {
     pub column: usize,
 }
 
+/// Types of symbols..
+#[derive(Clone, Copy)]
+pub enum SymbolType {
+    Unknown,
+    Function,
+    Variable,
+}
+
+/// The context of an address finding request.
+///
+/// This type passes additionl parameters to resolvers.
+#[doc(hidden)]
+pub struct FindAddrOpts {
+    /// Return the offset of the symbol from the first byte of the
+    /// object file if it is true. (False by default)
+    offset_in_file: bool,
```

Comment:
this is cheap and easy to calculate, why is this an option? Just always return this and let user ignore it if they are not interested

Reply:
Agree!